core/store/migrate/migrations/0081_unconsumed_log_broadcasts.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+ALTER TABLE log_broadcasts ADD COLUMN updated_at timestamp with time zone NOT NULL;
+DROP INDEX IF EXISTS log_consumptions_unique_v2_idx;
+CREATE UNIQUE INDEX log_broadcasts_unique_idx ON log_broadcasts(job_id, block_hash, log_index, evm_chain_id);
+CREATE TABLE log_broadcasts_pending (
+    evm_chain_id numeric(78,0) REFERENCES evm_chains (id) DEFERRABLE INITIALLY IMMEDIATE PRIMARY KEY,
+    block_number int8,
```

Comment:
Why is the block number so tiny? And when would it be nil?

Reply:
I based it on the existing `log_broadcasts` table, but it appears to be standard across many other tables as well in the current schema.

Reply:
We set null if/when the pool is empty after a send.

Reply:
its an 8-byte int? agreed it couldn't be not null though